index.html

```diff
@@ -4,7 +4,76 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="css/style.css" />
+
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="logo">
+        <a href="#">
+          <h1>My Logo</h1>
+        </a>
+      </div>
+      <div>
+        <nav>
+          <span class="bars">
+            <i class="fa-solid fa-bars"></i>
+          </span>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="hero">
+        <div>
+          <h1 class="hero-name">I'm Mariane</h1>
+        </div>
+        <div>
+          <h2 class="hero-msg">Glad to see you!</h2>
+        </div>
```

Comment:
Just like the previous reviewer said, kindly increase the font size to meet up with the Figma template. 🥺 

| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="382" alt="image" src="https://user-images.githubusercontent.com/73854999/221201990-2b75a4be-65e2-4408-96f7-3b654c93ebfd.png"> |<img width="382" alt="image" src="https://user-images.githubusercontent.com/73854999/221202176-fea95a76-5e5d-4829-b312-5004d0ea9877.png">|